pkg/types/vulnerability.go

```diff
@@ -15,3 +16,17 @@ type DetectedVulnerability struct {
 
 	types.Vulnerability
 }
+
+// Less returns is vulnerability is less than other vulnerability
```

Comment:
We already defined `Less`, so what if adding `Len` and `Swap` so that we can use `sort.Sort`?
https://yourbasic.org/golang/how-to-sort-in-go/#sort-custom-data-structures